src/components/SideBar.vue

```diff
@@ -1,5 +1,10 @@
 <script setup>
 import LogoBlock from '../components/icons/LogoBlock.vue';
+import IconEvents from '../components/icons/IconEvents.vue';
+import IconHome from '../components/icons/IconHome.vue';
+import IconPrefs from '../components/icons/IconPrefs.vue';
+import IconSports from '../components/icons/IconSports.vue';
+import IconTeams from '../components/icons/IconTeams.vue';
```

Comment:
When listing a bunch of imports like this, let's get them all in alphabetical order by the import name.

Reply:
this is alphabetical right?

I remembered you mentioning it, so tried to make sure to do it :)

Reply:
The element above them is `LogoBlock`